src/genophenocorr/model/_variant_effects.py

```diff
@@ -74,3 +74,11 @@ def curie(self) -> str:
 
     def __str__(self) -> str:
         return self.name.lower()
+
```

Comment:
In Python, `Enum` members are singletons, therefore, `__eq__` and `__hash__` should not be overriden.